glancesync/glancesync_ami.py

```diff
@@ -42,19 +42,18 @@
 _logger = logging.getLogger('glancesync')
 
 
-def get_master_region_dict(image_list):
-    """Convert the image list to a dictionary indexed by name and also
-    replace at parameters kernel_id and ramdisk_id the UUID with the name of
+def clean_ami_ids(image_dict):
+    """replace at parameters kernel_id and ramdisk_id the UUID with the name of
```

Comment:
replace at kernel_id and ramdisk_id parameters the UUID with the name of the image.